egs/wsj/s5/steps/dict/train_g2p.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# Copyright 2014  Johns Hopkins University (Author: Yenda Trmal)
+# Apache 2.0
+
+# Begin configuration section.  
+iters=5
+stage=5
+encoding='utf-8'
+remove_tags=true
+only_words=true
+icu_transform="Any-Lower"
+cmd=run.pl
```

Comment:
I just talked with Dan and let's go with ignoring the normalization issues and let's make it user's job.. The same with the apply_g2p.sh
That will simplify the g2p scripts considerably